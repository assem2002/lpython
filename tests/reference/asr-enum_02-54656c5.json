{
    "basename": "asr-enum_02-54656c5",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/enum_02.py",
    "infile_hash": "8ef155d733622824b261f286d72fb1be235bc6e73586a7acc02906d3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-enum_02-54656c5.stderr",
    "stderr_hash": "3b452b5d2b62f231751fd906f2105235733c122d37b6780b8fc72ab3",
    "returncode": 2
}