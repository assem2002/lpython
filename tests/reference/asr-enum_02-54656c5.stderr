semantic error: Enumerations with non-integer or non-unique integer values cannot interoperate with C code.
 --> tests/errors/enum_02.py:5:1 - 8:17
  |
5 |    class Color(Enum):
  |    ^^^^^^^^^^^^^^^^^^...
...
  |
8 |        BLUE: i32 = 2
  | ...^^^^^^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
